### PR TITLE
Move stuff to make it available for salt-ssh

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
Move some stuff from call_high to call_chunks so that it is available to salt-ssh.

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/39239

### Tests written?

No

### CODE REVIEW

File: salt/state.py [hunk 1]
```diff
@@ -2455,35 +2479,11 @@ def call_high(self, high, orchestration_jid=None):
...
-                        self.__run_num += 1
-                        chunks.remove(low)
-                        break
-
         # If there are extensions in the highstate, process them and update
         # the low data chunks
         if errors:
             return errors
-        ret = dict(list(disabled.items()) + list(self.call_chunks(chunks).items()))
+	ret = self.call_chunks(chunks)
```
Comment (s0undt3ch) [inline salt/state.py]: Missing a space? Tabs?

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: salt/state.py (hunk 1 above)
Comment (gtmanfred) [inline salt/state.py]: I swear, I am just going to delete my .vimrc tonight.

Comment (s0undt3ch) [inline salt/state.py]: :)

**Review by s0undt3ch — APPROVED** [review]
Test failure is not related.

**Pull request merged**